### DISCUSSION TITLE
minio configure tier timeout

### DISCUSSION THREAD

**Discussion by adeesh-vunet** [minio/minio#17514] [Q&A] [answered]
### Timeout when configuring remote tier
I'm trying to configure remote tier (referring doc : [minio](https://min.io/docs/minio/linux/administration/object-management/transition-objects-to-minio.html#minio-lifecycle-management-transition-to-minio-permissions)). But in my case that corresponding tier bucket is in the same cluster. When I do this, It would just timeout. 
Parameters I passed in UI are : 
![minio](https://github.com/minio/minio/assets/128485221/2ca20169-378b-4151-b230-102f99376e47)
It completely froze after I clicked the submit button. 

### I have added necessary permission for the access key and secret key. 
`{
 "Version": "2012-10-17",
 "Statement": [
  {
   "Effect": "Allow",
   "Action": [
    "admin:*"
   ]
  },
  {
   "Effect": "Allow",
   "Action": [
    "kms:*"
   ]
  },
  {
   "Effect": "Allow",
   "Action": [
    "s3:*"
   ],
   "Resource": [
    "arn:aws:s3:::*"
   ]
  },
  {
   "Sid": "EnableRemoteTierManagement",
   "Effect": "Allow",
   "Action": [
    "admin:ListTier",
    "admin:SetTier"
   ]
  },
  {
   "Sid": "EnableLifecycleManagementRules",
   "Effect": "Allow",
   "Action": [
    "s3:GetLifecycleConfiguration",
    "s3:PutLifecycleConfiguration"
   ],
   "Resource": [
    "arn:aws:s3:::*"
   ]
  },
  {
   "Effect": "Allow",
   "Action": [
    "s3:ListBucket"
   ],
   "Resource": [
    "arn:aws:s3:::test-bucket"
   ]
  },
  {
   "Effect": "Allow",
   "Action": [
    "s3:GetObject",
    "s3:PutObject",
    "s3:DeleteObject"
   ],
   "Resource": [
    "arn:aws:s3:::test-bucket/*"
   ]
  }
 ]
}`

I have also tried with cli using mc, I'm getting  timeout. 

Can some please comment on this behaviour

**Comment by harshavardhana** [comment, marked as answer]
You can't configure tier to be same cluster - we will improve and throw an error message.
